From 023d0adcb43db65e64841af6349a0326d2cb1e14 Mon Sep 17 00:00:00 2001
From: Dan Voiculeasa <dan.voiculeasa@windriver.com>
Date: Tue, 19 Apr 2022 11:58:01 +0300
Subject: [PATCH 1/2] dhclient: dhcp6 set hostname

The dhcp upstream commit 9261cb141 introduces a function
set_hostname() set host name, and the commit also implements
the DHCPv6 Handlers, so insert the set_hostname to the
BOUND6|RENEW6|REBIND6 implementation.

Signed-off-by: Yue Tao <yue.tao@windriver.com>
---
 debian/dhclient-script.linux | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/debian/dhclient-script.linux b/debian/dhclient-script.linux
index f9b734a..895116c 100644
--- a/debian/dhclient-script.linux
+++ b/debian/dhclient-script.linux
@@ -393,6 +393,8 @@ case "$reason" in
         ;;
 
     BOUND6|RENEW6|REBIND6)
+        set_hostname
+
         if [ "${new_ip6_address}" ]; then
             # set leased IP
             ip -6 addr add ${new_ip6_address} \
-- 
2.30.0

